### PR TITLE
Add Tables and Tests for author  and game class

### LINKED ISSUE #13: [2pt][Student3]: Add unit tests
- Add unit tests for all implemented methods.

### LINKED ISSUE #14: [1pt][Student3]: Add database "games" and "authors" tables
- Create a schema.sql file with tables that will be analogical to the structure of the classes that you created:
- games table (add all properties and associations from the parent Item class as table columns)
- authors table

### PR DESCRIPTION
**Pull request by camilevahviraki**
### In this branch we did the following:

- Add unit tests for `Game` and `Author` class.
- Add tables `Game` and `Author` tables in `schemas.sql`.

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -0,0 +1,21 @@
+CREATE DATABASE my_catalog;
+
+CREATE TABLE author(
+  ID SERIAL PRIMARY KEY,
+  first_name VARCHAR(30),
+  last_name VARCHAR(30),
+);
+
+CREATE TABLE book(
```
Comment (ha-manel) [inline schema.sql]: The table is called game, kindly correct that mistake 😅

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️ 
Hey @camilevahviraki great job so far, you still need to implement a tiny change before you can merge.

File: schema.sql (hunk 1 above)
Comment (camilevahviraki) [reply schema.sql]: :sweat_smile: What a mistake

**Review by ha-manel — APPROVED** [review]
# STATUS: APPROVED ✅ 
### Great job @camilevahviraki. It's time to merge 🎉

**Pull request merged**